docs/md/docs/2020-11-02-turn-the-senior-around.md

```diff
@@ -0,0 +1,68 @@
+# Turn The Senior Around
+
+As engineering managers we take many considerations into account, two of the main ones are:
```

Comment:
This sentence requires a bit of punctuation and capital letters :)
"As Engineering Managers, we take many considerations into account. "